pylmod/gradebook.py

```diff
@@ -76,6 +76,25 @@ def unravel_sections(section_data):
                 sections.append(section)
         return sections
 
+    @staticmethod
+    def unravel_staff(staff_data):
```

Comment:
If we were to change this to _unravel_staff(), we'd also want to change unravel_sections() to _unravel_sections().

And, "Yes", it is limited to unraveling staff.